docs/skopeo-copy.1.md

```diff
@@ -56,6 +56,10 @@ If the authorization state is not found there, $HOME/.docker/config.json is chec
 
 Existing signatures, if any, are preserved as well.
 
+**--dest-compress-format** _format_ Specifies the compression format to use.
+
+**--dest-compress-level** _format_ Specifies the compression level to use.
```

Comment:
This needs to be much clearer about what values are acceptable (and useful), and maybe what the default is.  (Consider improving the `--help` output similarly as well.)

Reply:
Also needs updated bash completions, I guess.